docs/units.rst

```diff
@@ -0,0 +1,16 @@
+=====
+Units
+=====
+
+mBuild automatically performs unit conversions in its reader and writer functions. 
+When working with an :py:class:`mbuild.Compound`, mBuild uses the following units:
+
++----------+---------+
+| Quantity |  Units  |
++==========+=========+
+| distance |    nm   |
++----------+---------+
+|   angle  | radians |
++----------+---------+
```

Comment:
Ah, we should include another section for the Lattice object, it uses `degrees` by default, since that is the common angle unit for crystallography.

Reply:
what angle unit are you using for the new Box class? We may want to include that too.

Reply:
Angle units for a box are also degrees.

Reply:
OK, I will make those notes--thank you :)